mithril-common/src/protocol/signer_builder.rs

```diff
@@ -29,6 +29,13 @@ pub enum SignerBuilderError {
     /// Error raised when the list of signers given to the builder is empty
     #[error("The list of signers must not be empty to create a signer builder.")]
     EmptySigners,
+
+    /// Error raised when the registration of a new party failed
+    #[error("Signer registration failed for party: '{party_id}'")]
+    Registration {
+        /// Party id of the signer
+        party_id: String,
```

Comment:
Same comment here

Reply:
Not useful anymore with the latest commit.